### ISSUE TITLE
Rova: KeyError: garbageTypeCode

### ISSUE THREAD

**Issue by don86nl** [home-assistant/home-assistant.github.io#31673]
### Feedback

All sensors are now unavailable. Log shows the following:

```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 599, in _async_add_entity
    await entity.async_device_update(warning=False)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1261, in async_device_update
    await hass.async_add_executor_job(self.update)
  File "/usr/local/lib/python3.12/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/rova/sensor.py", line 116, in update
    self.data_service.update()
  File "/usr/src/homeassistant/homeassistant/util/__init__.py", line 190, in wrapper
    result = method(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/rova/sensor.py", line 135, in update
    items = self.api.get_calendar_items()
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/rova/rova.py", line 80, in get_calendar_items
    garbage_type = item["garbageTypeCode"].upper()
                   ~~~~^^^^^^^^^^^^^^^^^^^
KeyError: 'garbageTypeCode'``
```

### URL

https://www.home-assistant.io/integrations/rova/

### Version

2024.1.*

**Comment by klaasnicolaas** [comment]
Hi there 👋

It seems like you want to report an issue with an Home Assistant integration, however, you have submitted documentation feedback instead. Please raise an issue in the Home Assistant core issue tracker instead:

https://github.com/home-assistant/core/issues

Thanks! 👍

./Klaas

**Comment by webbertakken** [comment]
FYI @klaasnicolaas you can also move issues between repositories, which is probably quicker and more search engine friendly than manually redirecting. I landed here by googling for the issue for example.

Found the follow-up in https://github.com/home-assistant/core/issues/112003 (marked as duplicate) and https://github.com/home-assistant/core/issues/110806 where the issue got solved.

TL;DR: 
- Bug was fixed in 2024.3.0. 
- Residual waste will start working again in 2024.3.1

**Comment by klaasnicolaas** [comment]
@webbertakken not everyone within the organization has the option to transfer issues to core 😉 We would therefore have to spend much more time on transferring issues while a user has already created his/her issue in the wrong place in the first place.

./klaas
